tr55/makeMiniAppTable.py

```diff
@@ -55,8 +63,11 @@
             }
         }
 
-        model_out = (simulate_day(cells, precip)
-                     ['unmodified']['distribution'].values()[0])
-        writer.writerow((precip, land_use,
-                         soil_type, model_out['et'],
-                         model_out['inf'], model_out['runoff']))
+        model_out = simulate_day(cells, precip)['unmodified']
```

Comment:
Ah I see. Originally we were pulling the values for the individual cell, but since the model is being run for only 1 cell we can pull the totals instead and get the same values.